docs/source/json-api/index.rst

```diff
@@ -892,27 +924,41 @@ Some notes on behavior:
    results if you walk the array forwards, backwards, or in random
    order.
 
-Fetch All Known Parties
-=======================
+Fetch by Key Contracts Stream
+-----------------------------
 
-- URL: ``/parties``
-- Method: ``GET``
-- Content: <EMPTY>
+- URL: ``/stream/fetch``
+- Scheme: ``ws``
+- Protocol: ``WebSocket``
 
-HTTP Response
--------------
+List currently active contracts that match one of the given ``templateId - key`` pairs, with continuous updates.
 
-- Content-Type: ``application/json``
-- Content:
+Similarly to `Contracts Query Stream`_, two subprotocols must be passed, as described in `Choosing a party
+<#choosing-a-party>`__.
+
+``application/json`` body must be sent first, formatted according to the following rule:
+
+.. code-block:: none
+
+    [
+        {"templateId": "<template ID 1>", "key": <key 1>}
```

Comment:
Can we please always put the commas at the end of the previous line. This looks even weird to me as a Haskeller.